code/modules/mob/dead/observer/orbit.dm

```diff
@@ -0,0 +1,90 @@
+/datum/orbit_menu
+	var/mob/dead/observer/owner
+	var/auto_observe = FALSE
+
+/datum/orbit_menu/New(mob/dead/observer/new_owner)
+	if(!istype(new_owner))
+		qdel(src)
+	owner = new_owner
+
+/datum/orbit_menu/tgui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/tgui_state/state = GLOB.tgui_observer_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if (!ui)
+		ui = new(user, src, ui_key, "Orbit", "title goes here", 800, 600, master_ui, state)
+		ui.open()
+
+/datum/orbit_menu/tgui_act(action, list/params, datum/tgui/ui)
+	. = ..()
+	if(.)
+		return
+
+	switch(action)
+		if ("orbit")
+			var/ref = params["ref"]
+
+			var/atom/movable/poi = (locate(ref) in GLOB.mob_list) || (locate(ref) in GLOB.poi_list)
+			if (poi == null)
+				. = TRUE
+				return
```

Comment:
Can this be shortened to `return TRUE`?

Reply:
It, technically, can.
The reason it is like this is because it's a /tg/ copypasta
Changing this to `return TRUE` unfortunately breaks the symmetry with the other `. = TRUE` lines around here..
I tried changing all of the `. = TRUE` to `return TRUE`, but I think I irrationally very slightly prefer the current version. But I'm happy to change that if anyone has particularly strong opinions here.